pkg/addon/uninstall.go

```diff
@@ -61,7 +61,8 @@ type deleteRHMIHandler struct {
 }
 
 var _ admission.Handler = &deleteRHMIHandler{}
-var _ admission.DecoderInjector = &deleteRHMIHandler{}
+
+//var _ admission.DecoderInjector = &deleteRHMIHandler{}
```

Comment:
why is this commented out?

Reply:
reasones:
1) Not used before
2) Doesn't exists. If it was used, we could try implement, but not used. Even more - workaround to replace missing function "DecoderInjector" is already implemented, but also not  used; it's `func (h *deleteRHMIHandler) InjectDecoder(d *admission.Decoder) error {
	h.decoder = d ...}

![Screenshot from 2024-06-09 10-50-53](https://github.com/integr8ly/integreatly-operator/assets/56625811/d41eebd6-ea9f-470e-8b71-309012ed60d9)

Thank you

Reply:
If it's used, please remove instead commenting out.